### PR TITLE
fix: Add CDS E2E tests to pipeline (GH-12732)

### LINKED ISSUE #12732: Migrate CDS e2es to the pipeline
Include CDS e2es and fixes to our pipeline, but should not a required check that blocks it

### PR DESCRIPTION
**Pull request by ShankariGiri-I**
The goal of this PR is to update and include CDS e2e tests to the travis pipeline so that its easy to test the integration library whenever a new version of spartacus needs to be released. 

The changes in this PR attempts to close: [GH-12732](https://github.com/SAP/spartacus/issues/12732)

### CODE REVIEW

**Review by bgambocjaviniar — COMMENTED** [review]
waiting on https://github.com/SAP/spartacus/pull/13685 to be merged before testing the e2e

**Comment by ShankariGiri-I** [comment]
Agree! However, the merchandising tests are being handled by team occtopus and they have time to fix this only in their upcoming sprint https://cxjira.sap.com/browse/OCC-6118. We thought we could get profile-tag tests to be working in the meantime.

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "start": "env-cmd --no-override -e dev,b2c,$SPA_ENV ng serve",
     "start:prod": "env-cmd --no-override -e dev,b2c,$SPA_ENV ng serve --configuration production",
     "start:pwa": "cd ./dist/storefrontapp/ && http-server --silent --proxy http://localhost:4200? -p 4200",
+    "start:cds": "env-cmd --no-override -e dev,b2c,cds,$SPA_ENV ng serve",
```
Comment (bgambocjaviniar) [inline package.json]: let's remove this

as you can always set cds to true before starting the application as follows

1 - go to `.env-cmd` and make B2C for `CX_CDS` true OR go to `environment.ts` and set remove the build.process and set it to true for CDS

File: .env-cmdrc [hunk 2]
```diff
@@ -5,6 +5,9 @@
   "ci": {
     "CX_BASE_URL": "https://20.83.184.244:9002"
   },
+  "cds": {
+    "CX_CDS": "true"
```
Comment (bgambocjaviniar) [inline .env-cmdrc:9]: add CX_CDS to false under b2c as well

**Review by bgambocjaviniar — CHANGES_REQUESTED** [review]

**Review by bgambocjaviniar — APPROVED** [review]

**Pull request merged**